quizApp.js

```diff
@@ -81,13 +170,15 @@ function onClickingShuffle(event) {
     }
   }
 }
-function displayQuestion(item) {
-  const section = document.getElementById("quiz-questions");
-  section.innerHTML = "";
+function displayQuestion(item, append = false) {
+  const section = document.getElementById("quizquestions");
```

Comment:
Your original `kebab-case` is more common practice in HTML

Reply:
Thank you for your feedback. I have implemented DRY principle and change the code into looping and followed kebab-case in html and css.